.github/workflows/codeql-analysis.yml

```diff
@@ -9,6 +9,7 @@ on:
 
 jobs:
   analyze:
+    if: github.repository == 'documenso/documenso'
```

Comment:
Any reason to not do this for forks?

Reply:
No specific reason, I think we can let it run on forks as well. I will send a commit to fix it :)